lib/stats/DefaultStatsFactoryPlugin.js

```diff
@@ -766,7 +766,7 @@ const SIMPLE_EXTRACTORS = {
 	},
 	moduleReason: {
 		_: (object, reason, { runtime }, { requestShortener }) => {
-			const depAsAny = /** @type {TODO} */ (reason.dependency);
+			const depAsAny = /** @type {Dependency|undefined} */ (reason.dependency);
```

Comment:
```suggestion
			const dep = reason.dependency;
			const moduleDep = dep instanceof ModuleDependency ? dep : null;
```

Reply:
Unfortunately, it does not work that way since Dependency does not have `userRequest`, see https://github.com/webpack/webpack/pull/11257/commits/18ded069f972cb1f19f4e9b83b9f27d0b36c8359